megazords/ios-rust/Cargo.toml

```diff
@@ -20,4 +20,6 @@ autofill = { path = "../../components/autofill" }
 push = { path = "../../components/push" }
 tabs = { path = "../../components/tabs" }
 places = {path = "../../components/places" }
+glean-core = { path = "../../components/external/glean/glean-core" }
+glean = { path = "../../components/external/glean/glean-core/rlb" }
```

Comment:
out of curiosity, why do we need to include both? do we need to expose the `rlb` at this layer?

Reply:
You may be correct here, I just added both, but I can try without the rlb dependency here.